### PR TITLE
Add gpg verification to workbench and connect

### PR DESCRIPTION
**Pull request by ianpittwood**
Adds the same gpg signature verification we use on package-manager/Dockerfile.bionic to all connect and workbench images.

### CODE REVIEW

**Comment by ianpittwood** [comment]
> @ianpittwood can we squash your commits into a single commit? it'll have a cleaner git history. otherwise lgtm!

@sarabrajsingh I'll use "Squash and merge" when closing the PR to get that outcome.

File: r-session-complete/Dockerfile.centos7 [hunk 1]
```diff
@@ -25,6 +25,10 @@ ARG RSW_DOWNLOAD_URL=https://s3.amazonaws.com/rstudio-ide-build/server/centos7/x
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN RSW_VERSION_URL=$(echo -n "${RSW_VERSION}" | sed 's/+/-/g') \
     && curl -o rstudio-workbench.rpm "${RSW_DOWNLOAD_URL}/${RSW_NAME}-${RSW_VERSION_URL}-x86_64.rpm" \
+    && gpg --keyserver keyserver.ubuntu.com --recv-keys 3F32EE77E331692F \
+    && gpg --export --armor 3F32EE77E331692F > rstudio-signing.key \
+    && rpm --import rstudio-signing.key \
+    && rpm -K rstudio-workbench.rpm \
     && yum install -y rstudio-workbench.rpm \
```
[the remark this answers is not in the document]
Comment (ianpittwood) [reply r-session-complete/Dockerfile.centos7:32]: I just went off our existing public docs for package validation in RHEL, but I can do more research.

[the remark this answers is not in the document]
Comment (atheriel) [reply r-session-complete/Dockerfile.centos7:32]: That's fine, then. Thanks!

**Review by atheriel — APPROVED** [review]

**Review by colearendt — APPROVED** [review]

**Comment by atheriel** [comment]
If the CI errors are spurious I don't see a reason not to merge this when you are ready, @ianpittwood.

**Comment by ianpittwood** [comment]
@atheriel yes, I think the failures were just one-offs. They succeeded on rerun.

We just need approval from @cm421 and @melissa-barca to merge now.

**Review by melissa-barca — APPROVED** [review]
LGTM!

**Pull request merged**